File: Src/API/API/Model/regio.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Model
{
    public class regio
    {
        public string naam { get; set; }
        public List<Locatie> locaties { get; set; }
        public int tijd { get; set; }
        public regio(string naam, List<Locatie> locaties, int tijd)
        {
            this.naam = naam;
            this.locaties = locaties;
            this.tijd = tijd;
        }
    }
}

File: MobApp/app/src/main/java/be/ap/eaict/geocapture/Model/Regio.java
package be.ap.eaict.geocapture.Model;

import java.util.List;

public class Regio {
    public int id;
    public String naam;
    public List<Locatie> locaties;
    public int tijd;

    public Regio(String naam, List<Locatie> locaties, int tijd)
    {
        this.naam = naam;
        this.locaties = locaties;
        this.tijd = tijd;
    }

    public String getNaam() {
        return naam;
    }

    public List<Locatie> getLocaties() {
        return locaties;
    }

    public int getTijd() {
        return tijd;
    }
}
